.vscode/settings.json

```diff
@@ -1,6 +1,34 @@
 {
+  "[python]": {
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": true
+    }
+  },
   "python.analysis.extraPaths": [
     "./src/package"
   ],
-  "python.defaultInterpreterPath": "./.venv/bin/python"
+  // Linux
+  "python.defaultInterpreterPath": "${workspaceFolder}/.venv/bin/python",
+  // Windows
+  // "python.defaultInterpreterPath": "${workspaceFolder}/.venv/Scripts/python.exe",
+  "python.envFile": "${workspaceFolder}/.env",
+  "python.linting.enabled": true,
+  "python.linting.banditEnabled": false,
```

Comment:
Yes, I disabled intentionally since bandit is not given as optional dev dependency:

https://github.com/jenstroeger/python-package-template/blob/7bf474b781efa1fb0e69dcf31f968675695f10bc/pyproject.toml#L44-L50

Reply:
Ah, neither is flake8. As such is it not consistent in the settings.json.

Reply:
Oh, correct, they’re managed by the pre-commit hooks:

https://github.com/jenstroeger/python-package-template/blob/7bf474b781efa1fb0e69dcf31f968675695f10bc/.pre-commit-config.yaml#L90-L99

and

https://github.com/jenstroeger/python-package-template/blob/7bf474b781efa1fb0e69dcf31f968675695f10bc/.pre-commit-config.yaml#L55-L65

The other tools (e.g. `pylint`) run from the local venv.

Reply:
Both bandit and flake8 are now disabled.